clubs_guides/models.py

```diff
@@ -1,3 +1,55 @@
 from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
 
-# Create your models here.
+
+@python_2_unicode_compatible
+class Category(models.Model):
+    """
+    Category a Mozilla Club's resource guide belongs to
+    """
+    name = models.CharField(max_length=200)
```

Comment:
200 might be a bit tricky for longer non-English names, but I guess we can roll with it until it proves to be a problem. We can always increase it I guess =D